libraries/bot-applicationinsights/src/main/java/com/microsoft/bot/applicationinsights/BotTelemetryClientImpl.java

```diff
@@ -107,21 +107,26 @@ public void trackAvailability(String name, OffsetDateTime timeStamp, Duration du
      * @param resultCode Result code of dependency call execution.
      * @param success True if the dependency call was handled successfully.
      */
+    @SuppressWarnings("checkstyle:ParameterNumber")
```

Comment:
Same

Reply:
The checkstyle complains about the amount of parameter in the constructor, we had to add that just like we did in [QnAMaker class](https://github.com/microsoft/botbuilder-java/blob/main/libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/dialogs/QnAMakerDialog.java#L489)

Reply:
Ack